<?xml version="1.0"?>

<launch>
<!--group ns="uav1"-->
    <arg name="self_system_id" default="1"/>
    <arg name="self_component_id" default="1"/>
    <arg name="target_system_id" default="1"/>
    <arg name="gcs_system_id" default="255"/>
    <arg name="init_position_x" default="0"/>
    <arg name="init_position_y" default="0"/>
    <arg name="init_position_z" default="0"/>
    <!--indoor path =1; outdoor = 0 -->
    <arg name="outdoor_indoor_flag" default = "0"/>
    <arg name="takeoff_acc" default = "0.5"/>
    <!--relative to ground takeoff hight -->
    <arg name="takeoff_hight" default = "3"/>
    <!--online path =1; offline = 0 -->
    <arg name="online_offline_flag" default = "0"/>  

   <node pkg="on_off_mission" type="on_off_mission" name="on_off_mission_node" output="screen">
    <!--rosparam command="load" file="$(find on_off_mission)/launch/waypoints_xyzy.yaml"/-->
	<!--param name="tm_outdoor_indoor_flag" default="$(arg outdoor_indoor_flag)"/-->
	<!--param name="tm_takeoff_acc" default="$(arg takeoff_acc)"/-->
	<!--param name="tm_takeoff_hight" default="$(arg takeoff_hight)"/-->
	<param name="tm_online_offline_flag" value="$(arg online_offline_flag)"/>
        <rosparam command="load" file="$(find config)/include/default.yaml" />
   </node>
<!--/group-->
</launch>
